src/atoms/Button/buttons.module.scss

```diff
@@ -1,6 +1,6 @@
 $info-color: color('primary-3');
 $action-color: color('primary-1');
-$selected-color: color('secondary-2');
```

Comment:
See my comment in `_colors.scss` for why this changed from `secondary-2` to `secondary-1` (it's the first bullet point)